### PR TITLE
8.0 auto refresh

### PR DESCRIPTION
**Pull request by oscarolar**

### CODE REVIEW

File: web_auto_refresh/web_auto_refresh.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            'temp', 'workflow', 'wizard', 'email_template', 'mass']:
+            action = model.env['ir.actions.act_window']
+            wired_count = 'stock.picking' == model._name and action.search_count(
+                [('res_model', '=', 'stock.picking.type'),
+                 ('auto_refresh', '>', '0')]) or 0
+
+            cnt = action.search_count([('res_model', '=', model._name), ('auto_refresh', '>', '0')])
+            if (wired_count or cnt) > 0:
+                bus = model.env['bus.bus']
+                bus.sendone('auto_refresh_kanban_list', model._name == 'stock.picking'  and 'stock.picking.type' or model._name)
```
Comment (nhomar) [inline web_auto_refresh/web_auto_refresh.py:89]: Why wired?

File: web_auto_refresh/web_auto_refresh.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+unlink_original = BaseModel.unlink
+
+
+@openerp.api.multi
+def unlink(self):
+    result = unlink_original(self)
+    auto_refresh_kanban_list(self)
+    return result
+BaseModel.unlink = unlink
```
Comment (nhomar) [inline web_auto_refresh/web_auto_refresh.py:73]: No monkey patch, this is a dead end.

File: web_auto_refresh/web_auto_refresh.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+write_original = BaseModel.write
+
+
+@openerp.api.multi
+def write(self, vals):
+    result = write_original(self, vals)
+    auto_refresh_kanban_list(self)
+    return result
+BaseModel.write = write
```
Comment (nhomar) [inline web_auto_refresh/web_auto_refresh.py:62]: No monkey Patch

Comment (oscarolar) [inline web_auto_refresh/web_auto_refresh.py:62]: I know, this is a test, if we decide to use it we can change it to a server action.

File: web_auto_refresh/web_auto_refresh.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+create_original = BaseModel.create
+
+
+@openerp.api.model
+@openerp.api.returns('self', lambda value: value.id)
+def create(self, vals):
+    record_id = create_original(self, vals)
+    auto_refresh_kanban_list(self)
+    return record_id
+BaseModel.create = create
```
Comment (nhomar) [inline web_auto_refresh/web_auto_refresh.py:51]: we have an approach in addons-vauxoo which can work with this. Once you measure the overload, we can review it in order to see if there is a better approach than this.

**Pull request closed without merging**